product_configurator_dynamaker/models/product.py

```diff
@@ -13,14 +13,50 @@ class DynamakerProductAttribute(models.Model):
 
     display_type = fields.Selection(selection_add=[('hidden_text', 'Hidden Text')])
 
+class Product(models.Model):
+    _inherit = 'product.product'
+
+    dynamaker_price = fields.Float(compute='_dynamaker_price')
+
+    def _dynamaker_price(self):
+        line_id = self.env.context.get('order_line_id')
+        custom_values = self.env.context.get('custom_values')
+        line = None
+        if line_id:
+            line = self.env['sale.order.line'].sudo().browse(line_id)
+        for product in self:
+            if not (line or custom_values):
+                product.dynamaker_price = product.list_price
+                continue
+            kw = {"width": 1000, "length": 1000, "edgeType": "standard", "thickness": 10}
```

Comment:
Put a ToDo: comment here on why we have this hardcoded and on what conditions we should remove it.